team/_posts/2022-9-26-frances-fuguy.md

```diff
@@ -11,6 +11,6 @@ github:
 scholar: 
 image: /assets/images/team/frances.jpg
 cv: /assets/pdfs/fuguy-CV.pdf
-alum: false
+alum: true
```

Comment:
We also need to update Frances' position as a graduate student at NAU. See Rebecca's profile for example.

Reply:
Will do.